TYP: ma.ones_like and ma.zeros_like not found by mypy because missing in ma/__init__.pyi

### Describe the issue:

mypy produces for both, `ma.ones_like` and `ma.zeros_like` the `error: Module has no attribute`. The apparent reason is that both are not listed in `ma/__init__.pyi`. Is this a simple omission? `ma/core.pyi` includes both, though, so `ma.core.ones_like` or `ma.core.zeros_like` work.

### Reproduce the code example:

```python
import numpy as np
import numpy.ma as ma
import numpy.ma.core as mac

x: ma.MaskedArray = ma.MaskedArray([1.0, 3.0, 5.0])
# x = ma.MaskedArray([1.0, 3.0, 5.0])
print(ma.ones_like(x))
print(ma.zeros_like(x))

print(np.ones_like(x))

print(mac.ones_like(x))
print(mac.zeros_like(x))
```


### Error message:

```shell
test.py:7: error: Module has no attribute "ones_like"  [attr-defined]
test.py:8: error: Module has no attribute "zeros_like"  [attr-defined]
Found 2 errors in 1 file (checked 1 source file)
```


### Python and NumPy Versions:

2.1.0
3.12.5 (main, Aug  7 2024, 00:00:00) [GCC 14.2.1 20240801 (Red Hat 14.2.1-1)]


### Type-checker version and settings:

mypy 1.11.2 (compiled: yes)

### Additional typing packages.

_No response_

--- a/numpy/ma/__init__.pyi
+++ b/numpy/ma/__init__.pyi
@@ -140,6 +140,7 @@ from numpy.ma.core import (
     nonzero as nonzero,
     not_equal as not_equal,
     ones as ones,
+    ones_like as ones_like,
     outer as outer,
     outerproduct as outerproduct,
     power as power,
@@ -178,6 +179,7 @@ from numpy.ma.core import (
     var as var,
     where as where,
     zeros as zeros,
+    zeros_like as zeros_like,
 )
 
 from numpy.ma.extras import (

--- a/numpy/ma/core.pyi
+++ b/numpy/ma/core.pyi
@@ -460,7 +460,9 @@ frombuffer: _convert2ma
 fromfunction: _convert2ma
 identity: _convert2ma
 ones: _convert2ma
+ones_like: _convert2ma
 zeros: _convert2ma
+zeros_like: _convert2ma
 
 def append(a, b, axis=...): ...
 def dot(a, b, strict=..., out=...): ...

